DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -52,5 +60,18 @@ public IEnumerable<DelegateCourseDetails> GetAllCoursesForDelegate(int delegateI
                 }
             );
         }
+
+        public bool RemoveDelegateFromCourse(int delegateId, int customisationId, RemovalMethod removalMethod)

```

Comment:
We should probably name this method something that indicates it's usage and the fact that it can return false and not actually mark a progress record as removed.